Octokit.Reactive/Clients/IObservableRepositoryCommentsClient.cs

```diff
@@ -13,81 +19,157 @@ public interface IObservableRepositoryCommentsClient
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
         /// <param name="number">The comment id</param>
-        /// <returns></returns>
+        /// <returns>A <see cref="IObservable{CommitComment}"/> of <see cref="CommitComment"/> for the specified repository.</returns>
```

Comment:
Rather than call this out everywhere for the file, I'll just come back and check after this comment is folded away